app/src/main/java/org/oppia/android/app/drawer/ExitProfileDialogFragment.kt

```diff
@@ -43,6 +53,15 @@ class ExitProfileDialogFragment : DialogFragment() {
       false
     )
 
+    val isAdministratorControlsSelected = args.getBoolean(
+      BOOL_IS_ADMINISTRATOR_CONTROLS_SELECTED_KEY,
+      false
+    )
+
+    val lastCheckedItemId = args.getInt(
+      INT_LAST_CHECKED_ITEM_KEY
+    )
```

Comment:
Convert this to single line:
`val lastCheckedItemId = args.getInt(INT_LAST_CHECKED_ITEM_KEY)`

Reply:
Done